draft-ietf-oauth-status-list.md

```diff
@@ -81,69 +89,137 @@ Relying Party:
 : An entity that relies on the Status List to validate the status of the Referenced Token. Also known as Verifier.
 
 Status List:
-: A bit array that lists the statuses of many Referenced Tokens.
+: An object in JSON or CBOR representation containing a bit array that lists the statuses of many Referenced Tokens.
 
 Status List Token:
 : A token in JWT or CWT representation that contains a cryptographically secured Status List.
 
 Referenced Token:
-: A token in JWT or CWT representation which contains a reference to a Status List Token. The information from the contained Status List may give a Relying Party additional information about up-to-date status of the Referenced Token.
+: A token in JWT or CWT representation which contains a reference to a Status List or Status List Token. The information from the contained Status List may give a Relying Party additional information about up-to-date status of the Referenced Token.
 
-# JSON Web Token Representation
+# Status List {#status-list}
 
-## Status List JWT Format and Processing Requirements {#jwt-status-list-format-and-processing}
+A Status List is a byte array that contains the statuses of many Referenced Tokens represented by one or multiple bits. A common representation of a Status List is composed by the following algorithm:
```

Comment:
With the proposal of #90 we would need to re-word this to be a bit more abstract right? Given the status list might take a different form?